circus-train-s3-s3-copier/src/main/java/com/hotels/bdp/circustrain/s3s3copier/aws/ExecutorServiceFactory.java

```diff
@@ -0,0 +1,43 @@
+/**
```

Comment:
Can we just have a simple test class for this?

Reply:
good catch - this is actually completely unused and unneeded now due to it being replaced by the lambda call. Removed.